src/NuGet.Jobs.GitHubIndexer/ReposIndexer.cs

```diff
@@ -17,7 +17,7 @@ namespace NuGet.Jobs.GitHubIndexer
 {
     public class ReposIndexer
     {
-        private const string WorkingDirectory = "work";
+        private static string WorkingDirectory = Path.Combine(Path.GetTempPath(), "NuGet.Jobs.GitHubIndexer");
```

Comment:
Can this be readonly?

```suggestion
        private static readonly string WorkingDirectory = Path.Combine(Path.GetTempPath(), "NuGet.Jobs.GitHubIndexer");
```

Supernit: generally we put `const` fields before `static` fields

Reply:
done